crates/subspace-farmer/src/multi_farming.rs

```diff
@@ -79,6 +80,7 @@ impl MultiFarming {
         best_block_number_check_interval: Duration,
         plot_sizes: Vec<u64>,
         new_plot: impl Fn(usize, PublicKey, u64) -> Result<Plot, PlotError> + Clone + Send + 'static,
+        start_farmings: bool,
```

Comment:
Now abstraction seems a bit broken if we have `MultiFarming` with farming disabled. It doesn't seems to make a lot of sense when combined together, does it?

Reply:
I would leave it under the same abstraction though, as it allows code reuse. Should I rename `MultiFarming` to smth else?

Reply:
Maybe, but I can't come up with much better name right now.

Reply:
I think this is just an exception for the benchmarking case. That's why this flag was hidden in the first place, so that you either call `new` or `benchmarking`.

Reply:
We can keep it for now then